Gemfile.lock

```diff
@@ -2141,8 +2141,8 @@ GEM
     jquery-ui-rails (7.0.0)
       railties (>= 3.2.16)
     json (2.5.1)
-    json-schema (4.3.1)
-      addressable (>= 2.8)
+    json-schema (5.0.0)
+      addressable (~> 2.8)
```

Comment:
@jayreddy519 @jacobkagon Same concern as the previous PR. We only have to update the `rexml` gem and its dependencies, not others.